Task1.py

```diff
@@ -0,0 +1,25 @@
+# Напишите программу, которая получает целое
+# число и возвращает его шестнадцатеричное
+# строковое представление. Функцию hex
+# используйте для проверки своего результата
+
+number = int(input('Enter a whole number: '))
+
+
+def get_hex(num: int) -> str:
+    result = ''
+    symbols = 'abcdef'
+    while num != 0:
+        remainder = num % 16
+        if remainder < 10:
+            result = str(remainder) + result
+        else:
+            result = symbols[remainder - 10] + result

```

Comment:
Если добавить все остальные символы в строку, то можно избавиться от условий.
```suggestion
    symbols = '0123456789abcdef'
    while num != 0:
        result = symbols[num % 16] + result
```